src/PointwiseFunctions/AnalyticData/Xcts/Binary.cpp

```diff
@@ -0,0 +1,111 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticData/Xcts/Binary.hpp"
+
+#include <cstddef>
+#include <utility>
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Elliptic/Systems/Xcts/Tags.hpp"
+#include "PointwiseFunctions/AnalyticData/Xcts/CommonVariables.tpp"
+#include "PointwiseFunctions/Elasticity/Strain.hpp"
+#include "PointwiseFunctions/GeneralRelativity/IndexManipulation.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "PointwiseFunctions/Xcts/LongitudinalOperator.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+
+/// \cond
+namespace Xcts::AnalyticData::detail {
+
+template <typename DataType>
+void BinaryVariables<DataType>::add_deriv_of_window_function(
+    const gsl::not_null<tnsr::ijj<DataType, Dim>*> deriv_conformal_metric)
+    const noexcept {
+  if (not falloff_widths.has_value()) {
+    return;
+  }
+  const auto& flat_metric =
+      get<Tags::ConformalMetric<DataType, 3, Frame::Inertial>>(flat_vars);
+  for (size_t i = 0; i < 3; ++i) {
+    for (size_t j = 0; j < 3; ++j) {
+      for (size_t k = 0; k <= j; ++k) {
+        for (size_t alpha = 0; alpha < 2; ++alpha) {
```

Comment:
maybe rename `alpha` to `object_index` since alpha is used for the lapse, and Greek indices are sometimes used in GR?